packages/inertia-react/index.d.ts

```diff
@@ -41,9 +41,9 @@ interface InertiaLinkProps {
   onSuccess?: () => void
 }
 
-type InertiaLink = React.FunctionComponent<
-  InertiaLinkProps & Omit<React.HTMLAttributes<HTMLElement>, 'onProgress'> & React.AllHTMLAttributes<HTMLElement>
->
+type InertiaLinkProps = InertiaLinkProps & Omit<React.HTMLAttributes<HTMLElement>, 'onProgress'> & React.AllHTMLAttributes<HTMLElement>
```

Comment:
Shouldn't this be `BaseInertiaLinkProps`?

```suggestion
type InertiaLinkProps = BaseInertiaLinkProps & Omit<React.HTMLAttributes<HTMLElement>, 'onProgress'> & React.AllHTMLAttributes<HTMLElement>
```

Reply:
Ok, I've changed it